docs/snapchat.md

```diff
@@ -0,0 +1,25 @@
+# Integrating the Snapchat Provider
+
+## Example
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+	services.AddAuthentication(options => /* Auth configuration */)
+			.AddSnapchat(options =>
+			{
+				options.ClientId = "my-client-id";
+				options.ClientSecret = "my-client-secret";
+			});
```

Comment:
Tabs make the formatting in the rendered document look less-good.

```suggestion
    services.AddAuthentication(options => /* Auth configuration */)
            .AddSnapchat(options =>
            {
                options.ClientId = "my-client-id";
                options.ClientSecret = "my-client-secret";
            });
```

![image](https://user-images.githubusercontent.com/1439341/185639139-bb494bb9-d85e-43c2-be3b-dec1960580a8.png)